### PR TITLE
fix: Resolve conflicting IPv6 and IPv4 addresses

### PR DESCRIPTION
**Pull request by rtaiello**
I tested with Fedora 36 and localhost is translated in ::: rather than 127.0.0.1. It might be better to change it with 127.0.0.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,9 +208,9 @@ PARTIES = 4  # number of parties that will be involved in the protocol, you can
 
 def setup_local_pool(server_port: int, ports: List[int]) -> Pool:
     pool = Pool()
-    pool.add_http_server(server_port, "localhost")
+    pool.add_http_server(server_port, "127.0.0.1")
```
Comment (ThomasTNO) [inline README.md]:
```suggestion
    pool.add_http_server(server_port)
```

File: README.md [hunk 2]
```diff
@@ -208,9 +208,9 @@ PARTIES = 4  # number of parties that will be involved in the protocol, you can
...
     for client_port in (port for port in ports if port != server_port):
-        pool.add_http_client(f"client{client_port}", "localhost", client_port)
+        pool.add_http_client(f"client{client_port}", "127.0.0.1", client_port)
```
Comment (ThomasTNO) [inline README.md]:
```suggestion
        pool.add_http_client(f"client{client_port}", "localhost", client_port)
```

**Review by ThomasTNO — CHANGES_REQUESTED** [review]
Thanks for catching this flaw in the documentation!

Made some suggestions before merging. Apart from those could you also modify line 262 in the README.md file to
```python
pool.add_http_server(server_port)
```
and the typo on line 175 to
```
### Advanced usage
```

Thanks again

**Comment by ThomasTNO** [comment]
> Thanks for catching this flaw in the documentation!
> 
> Made some suggestions before merging. Apart from those could you also modify line 262 in the README.md file to
> 
> ```python
> pool.add_http_server(server_port)
> ```
> 
> and the typo on line 175 to
> 
> ```
> ### Advanced usage
> ```
> 
> Thanks again

@rtaiello if you modify these I'll merge the PR.

**Comment by rtaiello** [comment]
Thanks @ThomasTNO I think I've done, I just commited them.

Let me know, thanks.

**Review by ThomasTNO — APPROVED** [review]

**Comment by ThomasTNO** [comment]
Incorporated, thanks a lot for the community contribution @rtaiello!

**Pull request merged**